addons/explosives/functions/fnc_addDetonateActions.sqf

```diff
@@ -32,20 +33,32 @@ private _explosivesList = [];
 
             _explosivesList pushBack _x;
 
-            _children pushBack
+            // Prevent consolidated detonate actions from having the same icon as consolidated place actions of the same explosive type
+            private _icon = if (
+                EGVAR(interact_menu,consolidateSingleChild) && 
+                {_detonator == GVAR(activeTrigger)} && 
+                {count (_result select {(_x select 4) == getText (_detonatorConfig >> QGVAR(triggerType))}) < 2}
+            ) then {
+                systemChat "true";
```

Comment:
This looks wrong.

Reply:
Damn it, I thought I caught all debug outputs, was a bit in a rush 🤦🏻‍♂️